adminer-editor/Dockerfile

```diff
@@ -17,15 +17,16 @@ RUN echo '@testing http://nl.alpinelinux.org/alpine/edge/testing' >> /etc/apk/re
         php7-session@testing \
         php7-mysqli@community \
         php7-pgsql@testing \
-        php7-pecl-mongodb@testing && \
+        php7-pecl-mongodb@testing \
+        dumb-init && \
```

Comment:
this looks like the `&& \` is on wrong line

Reply:
it's not, it's on the last line of the list of packages for `apk add` as it should

Reply:
Yeah sorry, I got confused.